feature/expenses/src/main/java/com/serranoie/app/feature/expenses/AddExpenseScreen.kt

```
@@ -675,39 +677,46 @@ fun MemberRow(
-                    textAlign = TextAlign.Center,
-                    style = MaterialTheme.typography.bodyMedium
-                )
+                        .weight(if (splitType == SplitType.PERCENTAGE) 0.25f else 0.4f)
+                        .alpha(if (member.included) 1f else 0.5f),
+                    contentAlignment = Alignment.Center
+                ) {
+                    AnimatedStrikethroughText(
+                        text = "${
+                            String.format(
+                                "%.2f", member.amount
+                            )
```

detekt (1.23.8)

[warning] String.format(
                                "%.2f", member.amount
                            ) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)